framework/src/mesh/MooseMesh.C

```diff
@@ -2677,6 +2700,9 @@ MooseMesh::init()
 
     getMesh().allow_renumbering(allow_renumbering_later);
     getMesh().skip_partitioning(skip_partitioning_later);
+
+    if (getParam<bool>("build_all_side_lowerd_mesh"))
```

Comment:
should you make a restartable attribute for this so the user does not need to specify it the second time?

Reply:
That does not seem possible in `MooseMesh`, right?

Reply:
why not?

Reply:
I did not see any call to `declareRestartableData` in there. I assume you mean that.

Reply:
eh, `MooseMesh` is indeed derived from `Restartable`. It might work.

Reply:
If something can be built quickly, do we want to rebuilt it to same some disk storage of the checkpoint files? Counter argument can be potentially missing the rebuilt. I feel we can rebuild here.

Reply:
Oh, I think I have a good reason to not use restartable data. It is because the data are restarted very late, too late for something like adding kernels (the one in Griffin) using this map.

Reply:
If that is the case it would be good to prove it with a test here in MOOSE

Reply:
Looks like one of the two maps has been built by `MooseMesh::cacheInfo`. The fix should be add the missing one there. I will push shortly.